docs/dev/build-instructions/README.md

```diff
@@ -12,6 +12,25 @@ The first step in building Mumble is to clone this repository and all the submod
 
 _Tip:_ You can also [build a specific version or commit](faq.md#build-a-specific-version-or-commit) of Mumble.
 
+
+## Using the BUILD_NUMBER variable
+
+We are currently using a [versioning scheme](https://www.mumble.info/blog/new-versioning-scheme/) that monotonically increases
+the "build" or "patch" component of the version string. This build number is specified via cmake as a variable e.g. ``-DBUILD_NUMBER=123``.
+Since we are exposing the complete version string in the Mumble UI as well as over the network protocol, it would be nice if the ``BUILD_NUMBER``
+variable is set correctly when building the binaries. This is especially relevant for **distribution package maintainers**.
+If not specified, the version string will end in ``.0``. This is also fine and not breaking anything, but it will be harder for
+use to determine the version of Mumble in bug reports and usage statistics.
+
+There exists a loose mapping between commit hash and ``BUILD_NUMBER``. If a ``BUILD_NUMBER`` exists for a given commit, it can be
```

Comment:
```suggestion
There exists a mapping between commit hash and ``BUILD_NUMBER``. If a ``BUILD_NUMBER`` exists for a given commit, it can be
```
The mapping is strict.